common/source/docs/common-esp32-telemetry.rst

```diff
@@ -0,0 +1,210 @@
+.. _common-esp32-telemetry:
+
+======================
+ESP32 WiFi telemetry
+======================
+
+The ESP32 are readily available Wi-Fi modules with full TCP/IP stack and
```

Comment:
advantages vs ESP-x 8226 for this application? which requires no programming, just wiring to autopilot

Reply:
The ESP32 is faster and does consume less power when WiFi encryption is enable. Most of the benefits the ESP32 has over the ESP8226 do not matter when is comes to this application. It all depends what kind of module you got lying around and is ready to use. The ESP8226 does not come pre-flashed - they are general purpose. I think there is a vendor that sells the 8266 module already pre-flashed.
The ESP32 has a lot of power left to be used for other applications - like doing additional sensors etc.